websocketOutportDriver/websocketOutportDriverNodePart.go

```diff
@@ -58,22 +59,38 @@ func NewWebsocketOutportDriverNodePart(args WebsocketOutportDriverNodePartArgs)
 }
 
 // SaveBlock will send the provided block saving arguments within the websocket
-func (o *websocketOutportDriverNodePart) SaveBlock(args *indexer.ArgsSaveBlockData) error {
-	return o.handleAction(args, outportSenderData.OperationSaveBlock)
+func (o *websocketOutportDriverNodePart) SaveBlock(args *outport.ArgsSaveBlockData) error {
+	var headerType string
+	if !check.IfNil(args.Header) {
+		headerType = reflect.TypeOf(args.Header).String()
```

Comment:
this is notoriously slow. Let's find another way here to provide the type

Reply:
changed